test/functional/p2p_invalid_messages.py

```diff
@@ -93,40 +101,47 @@ def test_msgtype(self):
             conn.sync_with_ping(timeout=1)
             self.nodes[0].disconnect_p2ps()
 
-    def test_large_inv(self):
-        conn = self.nodes[0].add_p2p_connection(P2PInterface())
-        with self.nodes[0].assert_debug_log(['Misbehaving', 'peer=4 (0 -> 20): message inv size() = 50001']):
-            msg = msg_inv([CInv(MSG_TX, 1)] * 50001)
-            conn.send_and_ping(msg)
-        with self.nodes[0].assert_debug_log(['Misbehaving', 'peer=4 (20 -> 40): message getdata size() = 50001']):
-            msg = msg_getdata([CInv(MSG_TX, 1)] * 50001)
-            conn.send_and_ping(msg)
-        with self.nodes[0].assert_debug_log(['Misbehaving', 'peer=4 (40 -> 60): headers message size = 2001']):
-            msg = msg_headers([CBlockHeader()] * 2001)
-            conn.send_and_ping(msg)
+    def test_oversized(self, msg_type, msg, size):
+        self.log.info("Test {} message of size {} is logged as misbehaving".format(msg_type, size))
+        with self.nodes[0].assert_debug_log(['Misbehaving', msg_type, str(size)]):
```

Comment:
splitting up msg_type and size could match them on different lines, while we want to match both on the same line

Reply:
Good point! Fixed. This made it apparent that in `net_processing.cpp` the debug logging for misbehaving from `headers` message size was a bit different from the equivalent misbehavior logging for `inv`, `getdata`, and `addr` message sizes. Fixed. If we don't want to touch it, we can also have each test pass in their own debug log message to test for.